tool/src/dart/mod.rs

```diff
@@ -748,6 +809,43 @@ impl<'a, 'cx> TyGenContext<'a, 'cx> {
         }
     }
 
+    /// Generates an FFI expression for a struct
+    fn gen_dart_to_c_for_struct_type(
+        &mut self,
+        dart_name: Cow<'cx, str>,
+        struct_borrow_info: Option<StructBorrowContext<'cx>>,
+    ) -> Cow<'cx, str> {
+        let mut params = String::new();
+        if let Some(info) = struct_borrow_info {
+            let edge_variable_expr = if info.is_method {
+                "edge_"
+            } else {
+                "...?append_array_for_"
```

Comment:
nit: camel case

Reply:
one day i'll be good at context switching casing norms when writing codegen